src/include/platform/NetworkCommissioning.h

```diff
@@ -218,6 +218,11 @@ class WirelessDriver : public Internal::BaseDriver
      */
     virtual CHIP_ERROR RevertConfiguration() = 0;
 
+    /**
+     * @brief Disconnect from network, but maybe it is not connected on media driver.
+     */
```

Comment:
We could guard this with `CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION`, but I think the default should be to return CHIP_ERROR_NOT_IMPLEMENTED.

Reply:
Done.